development-vm/Procfile

```diff
@@ -185,3 +185,5 @@ search-api-govuk-index-listener:  govuk_setenv search-api ./run_in.sh ../../sear
 search-api-bulk-reindex-listener: govuk_setenv search-api ./run_in.sh ../../search-api    bundle exec rake message_queue:bulk_insert_data_into_govuk
 # sidekiq-monitoring for search-api uses 3234
 content-data-api:      govuk_setenv content-data-api ./run_in.sh ../../content-data-api bundle exec rails server -p 3235
+content-publisher-sidekiq:               govuk_setenv content-publisher ./run_in.sh ../../content-publisher    bundle exec sidekiq -C ./config/sidekiq.yml
+# sidekiq-monitoring for content-publisher uses 3236
```

Comment:
Can we reserve ports 3237 and 3238 too for content publisher byebug? as per: https://github.com/alphagov/content-publisher/blob/master/Procfile.dev

I didn't realise someone had grabbed 3235 when I wrote those procfile changes so I'll have to up the port numbers by 1.

Reply:
Sure no worries.  I had reserved 3235, but by the time I opened the PR it had been taken!

Reply:
ha oh good, that makes me feel better about getting mine wrong 😁